README.rst

```diff
@@ -34,6 +34,22 @@ Since version 2.6 we introduce new ``--isort-show-traceback`` option.
 It is used to show verbose multi-line output from ``isort``.
 By default it is turned off.
 
+Error codes
+-------
```

Comment:
Could you make the underline match the title?

Reply:
@gforcada Sorry, did not understand you. Underline match the title?

Reply:
What you mean?

Reply:
I mean that the number of dashes match the length of the title above the dashes, i.e.:
```
Error codes
-----------
```

That would produce an RST syntax error when releasing to pypi for example.

Reply:
Oh, sure. Yes, I will do it.

Reply:
@gforcada fixed.